src/core/utils/url.js

```diff
@@ -23,11 +28,60 @@ export function removeUndefinedProps(object: Object): Object {
 export function addQueryParams(
   urlString: string,
   queryParams: { [key: string]: ?string | number } = {},
+  _config: typeof config = config,
 ): string {
+  let adjustedQueryParams = { ...queryParams };
+
+  if (
+    _config.get('enableFeatureUseUtmParams') &&
+    typeof queryParams.src !== 'undefined'
+  ) {
+    adjustedQueryParams = {
+      ...queryParams,
+      // Use UTM parameters instead of `src`, according to the PRD.
+      utm_source: DEFAULT_UTM_SOURCE,
+      utm_medium: DEFAULT_UTM_MEDIUM,
+      utm_content: queryParams.src,
```

Comment:
I find this a bit confusing, and I commented about it in the cases where we are passing `utm_content` in as `src`. I'm not a huge fan of reusing `queryParams.src` like this. Can you think of a different way of making this work by allowing either `utm_content` or `src` to be passed? Considering that we're only calling it this way a couple of times, it doesn't seem like it would be onerous on the caller.

Reply:
That's because of the feature flag. We won't keep that once we use UTM params everywhere. I don't really want to make this patch even bigger by not using the feature flag in the common functions (which I did here).